source/kraken/worker.cpp

```diff
@@ -415,7 +417,14 @@ pbnavitia::Response Worker::proximity_list(const pbnavitia::PlacesNearbyRequest
                                            const boost::posix_time::ptime& current_datetime) {
     const auto data = data_manager.get_data();
     type::EntryPoint ep(data->get_type_of_id(request.uri()), request.uri());
-    auto coord = coord_of_entry_point(ep, data);
+    type::GeographicalCoord  coord;
+    try{
+        coord = coord_of_entry_point(ep, data);
+    }catch(const std::exception& e) {
```

Comment:
catching such an anonymous exception will cause problem. I'm not found of this catch everywhere, but at least it's cleaner with a dedicated exception.